src/commands/php-errors.php

```diff
@@ -59,9 +59,9 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
         foreach( $pressable_sites->data as $_pressable_site ) {
             if ( ! empty( $site_domain ) && $site_domain !== $_pressable_site->url ) {
                 continue;
-            } else {
-                $pressable_site = $_pressable_site;
             }
+
+			$pressable_site = $_pressable_site;
```

Comment:
Should the indentation align with the above `if` statement?

Reply:
Hm looks good in my editor. Probably because of tabs/spaces. Will fix this.